### ISSUE TITLE
Docbuild fails because of missing fplll/sieve/sieve_gauss.h

### ISSUE THREAD

**Issue by jdemeyer** [fplll/fpylll#98]
Building the docs on ReadTheDocs is failing (since a while) with
```
build/src/fpylll/fplll/integer_matrix.cpp:569:37: fatal error: fplll/sieve/sieve_gauss.h: No such file or directory
```

**Comment by jdemeyer** [comment]
It seems that `fplll` does not install this header file. So either:

1. it is a bug in `fplll` that it doesn't install this header

or

2. it is a bug in `fpylll` that it wants to use a private `fplll` header

**Comment by malb** [comment]
This was fixed in commit fplll/fplll@2c9c9a25872500a5ddeb685b8448473a7885b4d1 we should probably cut a release, though.

**Comment by jdemeyer** [comment]
> we should probably cut a release, though.

+1 otherwise it's hard to use on ReadTheDocs
